libs/application/templates/rental-agreement/src/forms/draft/rentalHousingSubsections/rentalHousingFireProtections.ts

```
@@ -5,30 +5,58 @@ import {
   buildHiddenInputWithWatchedValue,
   buildRadioField,
+  buildAlertMessageField,
+  getValueViaPath,
 } from '@island.is/application/core'
 import { Routes } from '../../../utils/enums'
+import * as m from '../../../lib/messages'
+import { getRentalPropertySize, getYesNoOptions } from '../../../utils/utils'
+import { shouldShowSmokeDetectorsAlert } from '../../../utils/conditions'
+import { PropertyUnit } from '../../../shared/types'
 import { housingFireProtections } from '../../../lib/messages'
 import { getYesNoOptions } from '../../../utils/utils'
```

Biome (2.1.2)

[error] Shouldn't redeclare 'getYesNoOptions'. Consider to delete it or rename it.

'getYesNoOptions' is defined here:


(lint/suspicious/noRedeclare)